scripts/release/README.md

```diff
@@ -23,17 +23,18 @@
 
 #### Native Modules
 
-* Native _modules_ published to Marketplace include Native Mobile Resources (named `mobile-resources-native` in code) and Nanoflow Commons (named `nanoflow-actions-native` in code). `nanoflow-actions-hybrid` is deprecated since MX9.
+-   Native _modules_ published to Marketplace include Native Mobile Resources (named `mobile-resources-native` in code) and Nanoflow Commons (named `nanoflow-actions-native` in code). `nanoflow-actions-hybrid` is deprecated since MX9.
```

Comment:
From where am I getting these changes? I just added a note at line number 33.

Reply:
it follows the README convention. I think it is fine.